edx_toggles/toggles/__init__.py

```diff
@@ -2,4 +2,4 @@
 Expose public feature toggle API.
 """
 from .internal.setting_toggle import SettingDictToggle, SettingToggle
-from .internal.waffle import WaffleFlag, WaffleFlagNamespace, WaffleSwitch, WaffleSwitchNamespace
+from .internal.waffle.legacy import WaffleFlag, WaffleFlagNamespace, WaffleSwitch, WaffleSwitchNamespace
```

Comment:
To enable expand/contract, I was thinking this version would contain both the current names and the legacy names, pointing to the same classes. This would enable the introduction of the legacy names in advance of breaking compatibility for the existing names, to remove the need for lock-step merges in the future.

I don't know if this code works, or if there is another way you'll need to import these, but something like:
```
LegacyWaffleFlag = WaffleFlag
LegacyWaffleFlagNamespace = LegacyWaffleFlagNamespace
LegacyWaffleSwitch = WaffleSwitch
LegacyWaffleSwitchNamespace = WaffleSwitchNamespace
```

Reply:
Right. I understand.

Reply:
May be obvious, but please add a comment as well.  Thanks.